src/main/java/com/michelin/ns4kafka/util/RegexUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.michelin.ns4kafka.util;
+
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/**
+ * Regex utils.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class RegexUtils {
+    /**
+     * Convert wildcard strings list to regex patterns list.
+     *
+     * @param wildcardStrings The list of wildcard strings
+     * @return A list of regex patterns
+     */
+    public static List<String> wildcardStringsToRegexPatterns(List<String> wildcardStrings) {
+        return wildcardStrings.stream()
```

Comment:
Could you add unit tests for this class? I think we should verify:
- For a given input, the output matches what it is expected.
- For the output regex, it matches to expected strings.

I think the following tests should be a good start:
- abc.my* / abc.myTopic ➡️ Yes
- abc.my?opic / abc.myTopic ➡️ Yes
- abc.my*opics / abc.myTopic ➡️ No, because 's' in the left value
- abc.my*opic / abc.myTopics ➡️ No, because 's' in the right value
- abc.my*opic / abc.myAbcdefghiTopic ➡️ Yes
- abc.myT*op?c / abc.myTopiiic ➡️ No, too much 'i'
- abc.myT*o?ic / abc.myTopic ➡️ Yes
- abc.myT*o*ic / abc.myTopic ➡️ Yes
- abc.my*?p*c / abc.myTopic ➡️ Yes
- abc.myTopic** / abc.myTopic ➡️ Yes

I think handling `*` (0 or n) and `?` (0 or 1) is a good first step.

Reply:
I think `?` is commonly used to match only 1 character (not 0)
Examples for microsoft: https://support.microsoft.com/en-us/office/examples-of-wildcard-characters-939e153f-bd30-47e4-a763-61897c87b3f4

Reply:
I added the filterByPattern method in RegexUtils to filter any string with regex patterns.
So we have the filter by wildcard by combining:
1) the conversion from wildcard to regex
2) the filter by regex pattern

I added:
- TU for filter from regex patterns
- functional tests for filter from wildcards